### PR TITLE
sci-libs/arpack: new package, add 3.8.0_p20220604

### PR DESCRIPTION
**Pull request by Alessandro-Barbieri**
EAPI 8
cmake
int64
python

Signed-off-by: Alessandro Barbieri <lssndrbarbieri@gmail.com>

### CODE REVIEW

File: sci-libs/arpack/arpack-3.8.0_p20220604.ebuild [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+	if use icbexmm; then
+		exeinto "/usr/libexec/${PN}/examples/MATRIX_MARKET"
+		doexe "${BUILD_DIR}/EXAMPLES/MATRIX_MARKET/arpackmm"
+		rm "${BUILD_DIR}/EXAMPLES/MATRIX_MARKET/arpackmm" || die
+		docinto examples
+		dodoc -r "${BUILD_DIR}/EXAMPLES/MATRIX_MARKET"
+	fi
+	if use python; then
+		mkdir -p "${D}/$(python_get_sitedir)" || die
+		mv "${ED}/usr/$(get_libdir)/pyarpack" "${D}/$(python_get_sitedir)" || die
```
Comment (AndrewAmmerlaan) [inline sci-libs/arpack/arpack-3.8.0_p20220604.ebuild:100]: Can we use `python_domodule` here?

**Comment by AndrewAmmerlaan** [comment]
Why are we adding this here and not to ::gentoo?

File: sci-libs/arpack/Manifest [hunk 2]
```diff
@@ -0,0 +1,3 @@
+DIST arpack-3.8.0_p20220604.gh.tar.gz 1012341 BLAKE2B 30f71c334f3ce8b24bce309cdfa5869fa4781baa1935dbd7772936f1f135e4d6a3b576f9fece8ba9ff48f053bac1d1328c4f5f5545cacaa568e1ea5695013e2f SHA512 f31c7bd45a82a8a64e1d041ca18eac9c50dac0ebecf845ad5e57c70d1c12ab36ab0630db224897bfe6181c0fff9152f1dea7b3a6a2eb53bfda2f2a590ca8a37b
+DIST tutorial.ps.gz 105599 BLAKE2B 99570e6a0cc1c020b9c271e8ce1cc224f69ff5333cc818c10b063dc8ffbaf66a9f5c6ede99541b4212286d422146b1c4e332eba7e1635c99efa0e4c2620f0630 SHA512 a7b8fefd1a572ed477ae208d49094c1c65c9368990d12de9b3528e33fed546c947659f74e54a810d04154b4a1b0f9a94bcba4db0cf59d379e20786a261066aa4
```
Comment (thesamesam) [inline sci-libs/arpack/Manifest:2]: Too generic.

**Comment by Alessandro-Barbieri** [comment]
> Why are we adding this here and not to ::gentoo?

Because int64 can break other packages

**Pull request closed without merging**